.ci/scripts/calculate_jobs.py

```diff
@@ -134,11 +134,6 @@ def set_output(key: str, value: str):
                 "sytest-tag": "testing",
                 "postgres": "postgres",
             },
-            {
-                "sytest-tag": "buster",
```

Comment:
Thanks for catching this!

I was looking for a source of truth on Debian 10 (Buster) including Python 3.7 which I think is -> https://packages.debian.org/buster/python3.7

But also found a blog post talking about it https://pythonspeed.com/articles/stop-using-python-3.7/ which is also nice to confirm :fast_forward: 

> Debian 10 (“Buster”) includes Python 3.7, but Debian 11 and 12 do not.